source/MaterialXTest/ShaderGen.cpp

```diff
@@ -21,31 +21,33 @@
 
 namespace mx = MaterialX;
 
-// Observer to add library markers to all nodes. Use the "library" meta-data attribute to do this,
-// until namespace specification is resolved.
-static std::string LIBRARY_ATTRIBUTE("library");
-class LibraryObserver : public mx::Observer
+void loadLibraries(const mx::StringVec& libraryNames, const mx::FilePath& searchPath, mx::DocumentPtr doc)
```

Comment:
New utility method for loading libraries.

Reply:
Maybe I should move this into MaterialXShaderGen/Util.h later since it could be useful for others (our LibraryManager for example)

Reply:
Yes. We'll at least need this for LookDevX so would be good to have a reusable routine for this.